README.md

```diff
@@ -4,6 +4,12 @@
 
 To add a conference to the list, please [contribute](#contributing)!
 
+## 2019
```

Comment:
@DevS1993 thanks for adding this!

However, the conferences are segregated by year. can you add this in a new file called 2019-conferences-list in https://github.com/nikhita/tech-conferences-india/tree/master/year-pages? I would prefer to keep the README showing conferences only for the current year.

Reply:
> I would prefer to keep the README showing conferences only for the current year.

Having said this, a note below the list of conferences for 2018 in README pointing a link to the 2019 would be great.